reVX/x_mission_paths/path_finder.py

```diff
@@ -0,0 +1,455 @@
+"""
+Calculate least cost paths from supply curve points to transmission features
+
+Mike Bannister
+4/13/2021
+"""
+import logging
+import numpy as np
+import pandas as pd
+from math import sqrt
+import matplotlib.pyplot as plt
+
+from skimage.graph import MCP_Geometric
+
+from .config import CELL_SIZE, CLIP_RASTER_BUFFER, MINIMUM_DIST_KM
+
+logger = logging.getLogger(__name__)
+
+
+class BlockedTransFeature(Exception):
```

Comment:
We have a module for custom exceptions: 
https://github.com/NREL/reVX/blob/main/reVX/utilities/exceptions.py